### ISSUE TITLE
Custom fonts

### ISSUE THREAD

**Issue by tawrahim** [neoneye/SwiftyFORM#19]
Would it be possible to extend this library to allow developers to specify custom fonts in the tableView cell?

**Comment by neoneye** [comment]
Theming is not something I have spent time on. I think it should be possible to style things with `UIAppearance`, but there are probably obstacles. I ought to make an example of how to do it.

What do you think?

**Comment by tawrahim** [comment]
Hmm so am not exactly sure with `UIAppearance ` I quickly tried this
`        UITableViewCell.appearance().textLabel?.font = UIFont(name: "Exo2.0-Regular", size: 15.0)
` in the `viewDidLoad` method and it did not work. I was looking at the code and notice at round [line 34](https://github.com/neoneye/SwiftyFORM/blob/9f5b5e13e31059c8cbaf5e48f568b2598f72a052/Source/Cells/TextViewCell.swift#L34) you set the font style. I think it would be great if we can make that settable.

**Comment by neoneye** [comment]
Yes, the `init` code currently sets the fonts. It doesn't have to be like this, but I'm not sure what is best practice with theming. Maybe it's `UIAppearance` or something new swift appoach. Maybe it would be better to extend the `CellModel` with theming properties.

**Comment by tawrahim** [comment]
I think extending the `CellModel` makes the most sense and looks very clean to me.

**Comment by neoneye** [comment]
Awesome @tawrahim, I would love to see a proposal from you.

**Comment by tawrahim** [comment]
Haha great I would see if I can squeeze in sometime to work on it. 
I also think if it is easier for you to just extend it, It would be awesome since you are most familiar with the code :smile:

**Comment by neoneye** [comment]
Your cocoapod contributions are much appreciated. Never give up.

When I'm not working, then I'm coding on various iOS projects. There is too little time in this world.

**Comment by tawrahim** [comment]
Ok let me see what I can do since I'm also heavily involved with this [project](http://www.thetripbuddyapp.com/) to get v2.0 released. Lets see what happens.

**Comment by neoneye** [comment]
Your project looks cool. Is it pure swift?

**Comment by tawrahim** [comment]
Yeah it is! And we are using your amazing library for some of our view
controllers.

-T

**Comment by neoneye** [comment]
There is a local phenomenon here in Denmark, it's [gomore](https://gomore.dk/rental?utm_source=google&utm_medium=cpc&utm_campaign=rental-demand). I don't know if it's big outside europa. It seems related 👍

**Comment by tawrahim** [comment]
It is a little bit different but we both embody the spirit of the sharing economy :small_blue_diamond: 
In any case I would try and see if I can make some progress with this issue this weekend. However if you do get it to work on it first that would be even cooler :smile:
